src/editors/content/question/variables/secondgeneration/ModuleEditor.controller.ts

```diff
@@ -9,6 +9,7 @@ import { Maybe } from 'tsmonad';
 
 interface StateProps {
   activeChild: Maybe<ContentElement>;
+  sidebarShown: boolean;
```

Comment:
Adding this prop for the editor to re-render when it changes. doesn't work perfectly but it seems to help